Livecheckables/nspr.rb

```diff
@@ -1,4 +1,4 @@
 class Nspr
-  livecheck :url   => "http://ftp.mozilla.org/pub/mozilla.org/nspr/releases/",
+  livecheck :url   => "https://ftp.mozilla.org/pub/mozilla.org/nspr/releases/",
```

Comment:
```suggestion
  livecheck :url   => "https://ftp.mozilla.org/pub/nspr/releases/",
```

This redirects to https://ftp.mozilla.org/pub/nspr/releases/, so we should update the URL to avoid the redirection.